core/spec.md

```diff
@@ -6788,6 +6800,53 @@ plural name for the collection in its defined case.
 A request to inline an unknown, or non-inlinable, attribute MUST generate an
 error ([invalid_data](#invalid_data)).
 
+#### Sort Flag
+
+When a request is directed at a collection of Groups, Resources or Versions,
+the `?sort` query parameter (flag) MAY be used to indicate the order in which
+the entities of that collection are to be returned (i.e. sorted).
+
+The format of the `?sort` query parameter is:
+
+```yaml
+?sort=ATTRIBUTE[=asc|desc]
+```
+
+Where:
+- `ATTRIBUTE` MUST be the name of one of the attributes defined in
+  collection's entities, using the same dot (`.`) notation specified for the
+  `?inline` flag to specify an attribute within an object. The attribute MUST
+  only reference an attribute within the top-level collection, it MUST NOT
+  attempt to traverse into a nested xRegistry collection even if that nested
+  collection is inlined.
+- An OPTIONAL "sort order" (`asc` for ascending, `desc` for descending) MAY
+  be specified to indicate whether the first entity in the returned collection
+  MUST be the "lowest" values (`asc`) or whether it MUST be the "highest"
+  value (`desc`). When not specified, the default value MUST be `asc`.
+
+If a server supports sorting and the attribute specified is valid, but the
+the server is unable to support sorting over it, then an error
+([unsupported_sort_attribute](#unsupported_sort_attribute)) MUST be generated.
+If the specified attribute is not valid then an error
+([invalid_sort_attribute](#invalid_sort_attribute)) MUST be generated.
+
+When [pagination](../pagination/spec.md) is used to return the results, but
+the `?sort` flag is not specified, then the server MUST sort the results on the
+entities' `SINGULARid` value in ascending order.
+
+Sorting of strings MUST be done in a case-insensitive way.
```

Comment:
Just another reminder that we need to tighten this up (and any other occurrences of "case-insensitive").

Reply:
yep - thanks for the reminder: https://github.com/xregistry/spec/issues/356